ts/src/test/Exchange/base/test.sharedMethods.ts

```diff
@@ -11,6 +11,10 @@ function isTemporaryFailure (e) {
     return (e instanceof OperationFailed) && (!(e instanceof OnMaintenance));
 }
 
+function isArgumentsRequiredException (e) {
```

Comment:
I don't think we need this method, encapsulates something very simple that we can do directly

Reply:
if i recall it correctly, the custom imports were specifically handled inside transpiler.js's `isSharedMethodTests` logic block, don't remind the reason. if it cant be left such, i'll revise it a bit later today